pluginconfig/commit.go

```diff
@@ -136,7 +136,9 @@ type CommitOffchainConfig struct {
 	// ChainFeeAsyncObserverSyncFreq defines how frequently the async chain fee observer should sync.
 	ChainFeeAsyncObserverSyncFreq time.Duration `json:"chainFeeAsyncObserverSyncFreq"`
 
-	// ChainFeeAsyncObserverSyncTimeout defines the timeout for a single sync operation (e.g. fetch token prices).
+	// ChainFeeAsyncObserverSyncTimeout defines the timeout for a single
+	// chain fee observation operation (e.g. fetch token prices).
+	// NOTE: It is also used when the async observer is disabled, while making the sync calls.
```

Comment:
As part of this work lets make sure we set this value appropriately in the configs in chainlink (I think in ocr3.go)